### PR TITLE
feat: fetch paginated tags

### LINKED ISSUE #12: Tags from Octokit are sorted alphabetically, possibly causing problems for this action
Hello, and thank you for this github action. 
I am not sure, but this line https://github.com/loopwerk/tag-changelog/blob/1b03d1a537bb49a99ca80ba8f5448be1ea4a0497/src/index.js#L43
Seems to take the last 2 tags but not among the 10 last tags ? 
When I run locally in a repository ( using `gh api /repos/<owner>/<repo>/tags | jq`), I get 10 tags but not the last ones. 

In one of our projects we have always the same changlelog generated. 

I don't have a large set of debugging available, so I am sorry by advance if this is an error. 
Thank you

### PR DESCRIPTION
**Pull request by ldicarlo**
Closes #12

### CODE REVIEW

**Comment by ldicarlo** [comment]
@loopwerk, would that be ok for you ? I tested locally, but I think to add effective tests we'd need to add tags in your repo.

**Comment by ldicarlo** [comment]
(there was also a security update with npm)

**Comment by ldicarlo** [comment]
Oups this cannot work as is

File: src/index.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ async function run() {
...
   // Find the two most recent tags
-  const { data: tags } = await octokit.rest.repos.listTags({
-    owner,
-    repo,
-    per_page: 10,
-  });
+  // Using https://octokit.github.io/rest.js/v19#pagination
+  const { data: tags } = await octokit.paginate("GET /repos/{owner}/{repo}/tags",
+    { owner, repo },
+    response => response.data.map(d => d.name)
```
Comment (kevinrenskers) [inline src/index.js]: The map needs to be removed, as the full tag object is needed.

Comment (ldicarlo) [reply src/index.js]: Yes I just realized that sorry, I fixed it

Comment (kevinrenskers) [reply src/index.js]: Nice find with the paginate method though!

**Review by kevinrenskers — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -40,11 +40,10 @@ async function run() {
   }
 
   // Find the two most recent tags
-  const { data: tags } = await octokit.rest.repos.listTags({
-    owner,
-    repo,
-    per_page: 10,
-  });
+  // Using https://octokit.github.io/rest.js/v19#pagination
+  const { data: tags } = await octokit.paginate("GET /repos/{owner}/{repo}/tags",
```
Comment (kevinrenskers) [inline src/index.js:44]: This actually needed to be changed to just `const tags = ...` but I fixed it.

v1.1.1 is released, thanks for the PR!

Comment (ldicarlo) [reply src/index.js:44]: Oh yes of course I wrote it good locally and then I failed the copy/paste .. 
Thank you !

**Pull request merged**